.github/workflows/test-coverage.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  push:
+    branches: [ "develop" ]
+  pull_request:
+    branches: [ "develop" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+    
+    - name: Build
+    - run: npm ci
+    - run: npm run build --if-present
+    
+    - name: Run tests with coverage
+    - run: npm test
+
+    - name: Code Coverage Report
+      uses: irongut/CodeCoverageSummary@v1.3.0
+      with:
+        filename: ./src/coverage/cobertura-coverage.xml
```

Comment:
Just to make sure, did you tested if the report is being generated in this folder?

Reply:
Yeah, just ran locally and double checked! 👍